### ISSUE TITLE
custom keyboard shortcuts: 

### ISSUE THREAD

**Issue by ilteris** [vikdevelop/SaveDesktop#223]
### The current problem or need

Hello, does the app allow custom keyboard shortcuts to be exported? 

I have some keyboard shortcuts I edit in setting > keyboard > keyboard shortcuts > view and customize shortcuts in gnome 44.5

also additional layout options in tweaks. I would love to be able to export them as well. 

Thank you for such a great app.  

### Describe the solution you'd like

Have an option in the app to be able to export keyboard shortcuts.

**Comment by vikdevelop** [comment]
Hello,
your custom keyboard shortcuts are saved in the Dconf database, specifically in the `/org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/` key. For showing it, you can save the configuration with the SaveDesktop app and then show the `dconf-settings.ini` file in the configuration archive, or use the "Dconf editor" app and find the key.

**Comment by ilteris** [comment]
Thank you for the response. Just to confirm I have to import `/org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/` directory into the SaveDesktop app manually?

**Comment by vikdevelop** [comment]
No, SaveDesktop during saving configuration, saves the settings from the Dconf database into the `dconf-settings.ini` text file that is in the saved configuration archive. In this file are saved settings of your keyboard shortcuts, which look in the following form:

```ini
[org/gnome/settings-daemon/plugins/media-keys/custom-keybindings/custom0]
binding='<Control><Alt>t'
command='cmd'
name='name'
```
This text file (dconf-settings.ini) also includes other settings of your desktop environment, such as your favorite apps on the dock, theme and icon name, and also settings from Gnome Tweaks and much more.
